OpenRA.Game/Graphics/Sheet.cs

```diff
@@ -141,42 +128,32 @@ public Bitmap AsBitmap(TextureChannel channel, IPalette pal)
 
 		public void CreateBuffer()
 		{
-			lock (textureLock)
-			{
-				if (data != null)
-					return;
-				if (texture == null)
-					data = new byte[4 * Size.Width * Size.Height];
-				else
-					data = texture.GetData();
-				releaseBufferOnCommit = false;
-			}
+			if (data != null)
+				return;
+			if (texture == null)
+				data = new byte[4 * Size.Width * Size.Height];
+			else
+				data = texture.GetData();
+			releaseBufferOnCommit = false;
```

Comment:
Newline before this line, please.
Sorry I can't comment on the logical changes here. :disappointed: